### PR TITLE
refactor(models): name file related path more specific

### LINKED ISSUE #96: Rename path related properties in core config
rename `coreconfig.persist.outputPath` to `coreconfig.persist.outputDir`
rename `coreconfig.plugins[0].runner.outputPath` to `coreconfig.plugins[0].runner.outputFile`

### PR DESCRIPTION
**Pull request by BioPhoton**
closes #96

### CODE REVIEW

File: packages/utils/src/lib/mock/schema-helper.mock.ts [hunk 1]
```diff
@@ -47,14 +47,14 @@ export function mockPluginConfig(opt?: {
...
+        `require('fs').writeFileSync('${outputDir}', '${JSON.stringify(
           audits.map(
             ({ slug }) =>
               mockAuditReport({ auditSlug: slug }) satisfies AuditReport,
           ),
         )}')`,
       ],
-      outputPath: outputPath,
+      outputFile: outputDir,
     },
```
Comment (matejchalk) [inline packages/utils/src/lib/mock/schema-helper.mock.ts:58]: Changing this variable to `outputDir` is misleading, should either be `outputFile` or stay as `outputPath`.

**Review by matejchalk — CHANGES_REQUESTED** [review]
One related naming issue that I just thought of - we have the `--configPath` CLI argument. I suppose we should change that too. Either to `--configFile` or even just `--config` would be fine. (Can be a follow-up PR if you prefer.)

File: packages/utils/src/lib/mock/schema-helper.mock.ts (hunk 1 above)
Comment (BioPhoton) [reply packages/utils/src/lib/mock/schema-helper.mock.ts:58]: nice catch. renamed to outputFile

File: packages/utils/src/lib/mock/schema-helper.mock.ts [hunk 2]
```diff
@@ -78,11 +78,11 @@ export function mockAuditConfig(opt?: {
 }
 
 export function mockPersistConfig(opt?: Partial<PersistConfig>): PersistConfig {
-  let { outputPath, format } = opt || {};
-  outputPath = outputPath || __outputPath__;
+  let { outputDir, format } = opt || {};
+  outputDir = outputDir || __outputFile__;
```
Comment (matejchalk) [inline packages/utils/src/lib/mock/schema-helper.mock.ts:82]: This looks like a mistake. Should be probably be `'tmp'`.

Comment (BioPhoton) [reply packages/utils/src/lib/mock/schema-helper.mock.ts:82]: done

Comment (matejchalk) [reply packages/utils/src/lib/mock/schema-helper.mock.ts:82]: Is it? I still see the same diff :confused: 

image

Comment (BioPhoton) [reply packages/utils/src/lib/mock/schema-helper.mock.ts]: now

File: packages/utils/src/lib/mock/schema-helper.mock.ts [hunk 3]
```diff
@@ -13,7 +13,8 @@ const __pluginSlug__ = 'mock-plugin-slug';
 const __auditSlug__ = 'mock-audit-slug';
 const __groupSlug__ = 'mock-group-slug';
 const __categorySlug__ = 'mock-category-slug';
-const __outputPath__ = 'tmp/out-execute-plugin.json';
+const __outputDir__ = 'tmp';
+const __outputFile__ = 'out-execute-plugin.json';
```
Comment (matejchalk) [inline packages/utils/src/lib/mock/schema-helper.mock.ts:17]: All files inside `tmp`, please :pray: 

```suggestion
const __outputFile__ = `${__outputDir__}/out-execute-plugin.json`;
```

**Review by matejchalk — APPROVED** [review]

**Pull request merged**